src/dmd/expressionsem.d

```diff
@@ -10062,3 +10193,38 @@ private bool checkFunctionAttributes(Expression exp, Scope* sc, FuncDeclaration
         return error;
     }
 }
+
+// Get source code for `expr`
```

Comment:
Why is this different from the code in hdrgen.d which turns ASTs back into strings?